tools/vmd/BUILD.bazel

```diff
@@ -0,0 +1,39 @@
+load("@rules_pkg//pkg:tar.bzl", "pkg_tar")
+
+exports_files(["ios_vm_test_runner.template.sh"])
+
+sh_binary(
+    name = "vmd",
+    srcs = ["vm_run.sh"],
+    data = [
+        # Note: there are a plethora of problems from doing a trival build of this
+        # depenency with Bazel. For now, vendor the binary (56kb)
+        #
+        # We could always make a certificate - this needs further auditing for
+        # production usage.
+        "bin/sshpass",
```

Comment:
Recently I had to add a new binary to add a test case and missing instructions on how to generate / where it was coming from made me waste some time figuring that out myself.

Can you share how these were generated or add a comment with links to where they're coming from exactly? Did you manually compile `tools/vmd/ssh_pass.c`, if that's the case using what invocation exactly (should we document it here?)?

Reply:
Good call about adding some notes on how to compile this 👍